theme/.eslintrc.js

```diff
@@ -16,6 +17,8 @@ module.exports = {
     'plugin:react/recommended',
     'plugin:jsx-a11y/recommended',
```

Comment:
Consider swapping this with [`plugin:github/react` ](plugin:github/react)[plugin:github/react](https://github.com/github/eslint-plugin-github/blob/main/lib/configs/react.js) which will pull in this plugin along with additional rules. This plugin comes with `eslint-plugin-github`.

Reply:
Addressed